draco/debug.py

```diff
@@ -0,0 +1,218 @@
+from functools import cached_property
+from typing import Iterable, NamedTuple
+
+import altair as alt
+import pandas as pd
+
+from .draco import Draco
+
+
+class DracoDebug:
+    __DEFAULT_DRACO__ = Draco()
+    __DEFAULT_FEATURE_NAMES__ = set(__DEFAULT_DRACO__.soft_constraint_names)
+    __DEFAULT_WEIGHTS__ = __DEFAULT_DRACO__.weights
+
+    def __init__(
+        self,
+        specs: dict[str, Iterable[str] | str],
+        draco: Draco | None = None,
```

Comment:
why don't we initialize Draco in the default argument here?

Reply:
then None would also not be needed anymore

Reply:
I just wanted to be structurally consistent with how I handled default value settings for mutable args such as `feature_names`, but I realise that using a default arg for `draco` would make more sense.